### ISSUE TITLE
Pull a list of tools available in Canvas and configure from this rather than configuring manually.

### ISSUE THREAD

**Issue by danie1zhang** [tl-its-umich-edu/canvas-app-explorer#9]

**Comment by zqian** [comment]
So here is the list of [Canvas External Tools](https://documentation.its.umich.edu/node/1222)

Does it meet the need of this issue? @jonespm

**Comment by jonespm** [comment]
No, the idea of this was to pull from the Canvas API (or similar source) like `/api/v1/accounts/1/external_tools `and populate some details in the local database. Like the tool name, Canvas ID, description, placements, icons etc. 

This could be done for just the root account initially but might be useful for sub accounts.

Currently everything has to be manually populated. This was created really early before any Canvas integration was available. 

I'm thinking this would either be done when an admin clicks a button to refresh the tools available. They would have to still go in and "enable" them but it could save some effort.
